### PR TITLE
solution

### PR DESCRIPTION
**Pull request by donizer**
- [DEMO LINK](https://donizer.github.io/layout_calendar/)
- [TEST REPORT LINK](https://donizer.github.io/layout_calendar/report/html_report/)

for some reason report doesn't creates after tests, so second link broken

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,5 +8,39 @@
   </head>
   <body>
     <h1>Calendar</h1>
+
```
Comment (volodymyr-soltys97) [inline src/index.html]:
```suggestion
```

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
   margin: 0;
 }
+
+.calendar {
+  $days: 31;
+  $parent: &;
+  $startDays: mon, tue, wed, thu, fri, sat, sun;
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss]: Move it to variables file

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
You need to remove the title, center the calendar on the page vertically and add a 1px indent between the days for a better look
image

This part of the task is incomplete
image

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome work!

**Pull request closed without merging**